frontend/src/bundles/common/components/tabs/tab.tsx

```diff
@@ -0,0 +1,55 @@
+import clsx from 'clsx';
+import React, { type KeyboardEvent, type ReactNode,useCallback, useState } from 'react';
+
+import styles from './style.module.scss';
+
+type TabProperties = {
+    children: ReactNode;
+};
+
+const Tab: React.FC<TabProperties> = ({ children }) => {
+    const [activeTab, setActiveTab] = useState<number>(0);
+
+    const handleTabClick = useCallback((index: number) => {
+        setActiveTab(index);
+    }, [setActiveTab]);
+
+    const handleKeyDown = (event: KeyboardEvent<HTMLDivElement>, index: number): void => {
```

Comment:
You would have not to use this event handler if there was `button` instead of `div`

Reply:
ok now we have buttons